demisto_sdk/commands/content_graph/objects/base_content.py

```diff
@@ -316,6 +325,45 @@ def from_path(
     def match(_dict: dict, path: Path) -> bool:
         pass
 
+    @property
```

Comment:
how does this affect the graph that gets built? We dont actually want this collected when building the graph. How can we mark this as a transient property? @ilaner

Reply:
it doesn't affect the graph, so it won't be created as a property of the content-item within the graph